requirements.txt

```diff
@@ -4,9 +4,9 @@ python-dateutil>=2.6.1
 requests>=2.0.0,<3
 promise>=2.0,<3
 shortuuid>=0.5.0
-six>=1.13.0
 psutil>=5.0.0
 sentry-sdk>=1.0.0
+six>=1.13.0
```

Comment:
I really hope we get back to the vendored libraries and do something about them so that we can ditch six for good. for example, instead of vendoring, fork the needed repo versions to our org, modify the necessary parts, and then depend on the forks.